<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1304</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.637, y2:1, stop:0 rgba(0, 0, 15, 255), stop:1 rgba(21, 39, 67, 255));
font: 11pt &quot;Umpush&quot;;
color: rgb(211, 215, 207);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="statusLabel">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0, 0);
color: white;
font: 75 36pt &quot;DejaVu Sans Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Standby</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 0, 0, 0);
font: 25 18pt &quot;Umpush&quot;;</string>
        </property>
        <property name="text">
         <string>Launch Control</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="Line" name="line">
      <property name="styleSheet">
       <string notr="true">color: rgba(200, 200, 255, 100);</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="countdownLbl">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0, 0);
font: 172pt &quot;Umpush&quot;;</string>
      </property>
      <property name="text">
       <string>T-05:00:00.0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0" rowspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="armBtn">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 100, 0, 0);
color: white;
outline: none;
font: 75 15pt &quot;Sans&quot;;</string>
        </property>
        <property name="text">
         <string>ARM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startCdnBtn">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 100, 0, 0);
font: 75 15pt &quot;Sans&quot;;
color: white;
outline: none;</string>
        </property>
        <property name="text">
         <string>START COUNTDOWN</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="haltBtn">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(200, 100, 0, 0);
color: white;
outline: none;
font: 75 15pt &quot;Sans&quot;;</string>
        </property>
        <property name="text">
         <string>HALT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resumeBtn">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 100, 0, 0);
color: white;
outline: none;
font: 75 15pt &quot;Sans&quot;</string>
        </property>
        <property name="text">
         <string>RESUME</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="abortBtn">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(150, 0, 0, 0);
color: white;
outline: none;
font: 75 15pt &quot;Sans&quot;;</string>
        </property>
        <property name="text">
         <string>ABORT</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <widget class="Line" name="line_2">
      <property name="styleSheet">
       <string notr="true">color: rgba(200, 200, 255, 100);</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="Line" name="line_3">
      <property name="styleSheet">
       <string notr="true">color: rgba(200, 200, 255, 100);</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QGroupBox" name="cfgBox">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
      </property>
      <property name="title">
       <string>Configuration</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="cfgPulseSpinbox">
         <property name="styleSheet">
          <string notr="true">background-color: darkgreen;
color: white;
</string>
         </property>
         <property name="prefix">
          <string>Pulse duration: </string>
         </property>
         <property name="suffix">
          <string> ms</string>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>100.000000000000000</double>
         </property>
         <property name="value">
          <double>5000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="cfgDurationSpinbox">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: darkgreen;
color: white;
</string>
         </property>
         <property name="wrapping">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
         <property name="prefix">
          <string>Countdown Duration: </string>
         </property>
         <property name="suffix">
          <string> seconds</string>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="maximum">
          <double>3599.000000000000000</double>
         </property>
         <property name="value">
          <double>15.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cfgApplyBtn">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(100, 200, 50, 255);
color: black;</string>
         </property>
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="11" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
